### PR TITLE
Network-performance comparison

### PR DESCRIPTION
**Pull request by rsevilla87**
Signed-off-by: Raul Sevilla <rsevilla@redhat.com>

### Description
Migration of the network-performance workload to the new comparison wrappers.

### CODE REVIEW

**Review by jtaleric — APPROVED** [review]
LGTM

File: workloads/network-perf/run_serviceip_network_test_fromgit.sh [hunk 1]
```diff
@@ -10,7 +10,8 @@ for pairs in 1 2 4; do
   if [[ $? != 0 ]]; then
     exit 1
   fi
-  assign_uuid
+  BASELINE_UUID=${BASELINE_SVC_UUID[${i}]}
+  COMPARISON_OUTPUT="${PWD}/service-${pairs}-pairs.csv"
   run_benchmark_comparison
 done
 generate_csv
```
Comment (Harshith-umesh) [inline workloads/network-perf/run_serviceip_network_test_fromgit.sh:17]: need to delete this line, generate_csv fn has been removed.

File: workloads/network-perf/run_serviceip_network_policy_test_fromgit.sh [hunk 2]
```diff
@@ -1,20 +1,22 @@
...
+  COMPARISON_OUTPUT="${PWD}/service-networkpolicy-${pairs}-pairs.csv"
   run_workload ripsaw-uperf-crd.yaml
   if [[ $? != 0 ]]; then
     exit 1
   fi
-  assign_uuid
+  BASELINE_UUID=${BASELINE_SVC_UUID[${i}]}
   run_benchmark_comparison
 done
 generate_csv
```
Comment (Harshith-umesh) [inline workloads/network-perf/run_serviceip_network_policy_test_fromgit.sh:22]: need to delete this line, generate_csv fn has been removed.

File: workloads/network-perf/run_pod_network_test_fromgit.sh [hunk 3]
```diff
@@ -1,15 +1,16 @@
...
   run_workload ripsaw-uperf-crd.yaml
   if [[ $? != 0 ]]; then
     exit 1
   fi
-  assign_uuid
+  BASELINE_UUID=${BASELINE_POD_UUID[${i}]}
+  COMPARISON_OUTPUT="${PWD}/pod-${pairs}-pairs.csv"
   run_benchmark_comparison
 done
 generate_csv
```
Comment (Harshith-umesh) [inline workloads/network-perf/run_pod_network_test_fromgit.sh:16]: need to delete this line, generate_csv fn has been removed.

File: workloads/network-perf/run_pod_network_policy_test_fromgit.sh [hunk 4]
```diff
@@ -10,7 +10,8 @@ for pairs in 1 2 4; do
   if [[ $? != 0 ]]; then
     exit 1
   fi
-  assign_uuid
+  BASELINE_UUID=${BASELINE_POD_UUID[${i}]}
+  COMPARISON_OUTPUT="${PWD}/pod-networkpolicy-${pairs}-pairs.csv"
   run_benchmark_comparison
 done
 generate_csv
```
Comment (Harshith-umesh) [inline workloads/network-perf/run_pod_network_policy_test_fromgit.sh:17]: need to delete this line, generate_csv fn has been removed.

**Pull request merged**